docker_push_latest_if_changed.py

```diff
@@ -19,8 +134,27 @@ def main(argv=None):
             '`--source` image.'
         ),
     )
-    parser.parse_args(argv)
-
+    parser.add_argument(
+        '-v',
+        '--verbose',
+        action='count',
+        default=0,
+        help=(
+            'Make output more verbose. Use twice for more verbosity.'
+        ),
+    )
+    parser.add_argument(
+        '--dry-run',
+        action='store_true',
+        help=(
+            "Run command, but don't actually push or tag images."
+        ),
```

Comment:
Oh yeah, I'll condense it. I still think there's some value in keeping the help message though since even though `dry-run` is a common term, for anyone uninitiated it could potentially be an ambiguous parameter name.